langchain/src/vectorstores/rockset.ts

```diff
@@ -0,0 +1,418 @@
+import { MainApi } from '@rockset/client';
+import type { CreateCollectionRequest } from "@rockset/client/dist/codegen/api.d.ts";
```

Comment:
I think you will need to add this path here:

https://github.com/langchain-ai/langchainjs/blob/main/langchain/scripts/check-tree-shaking.js#L24